server/src/main/java/org/elasticsearch/rest/action/search/RestSearchAction.java

```diff
@@ -95,6 +95,11 @@ public List<Route> routes() {
         );
     }
 
+    @Override
+    public Set<String> supportedCapabilities() {
+        return SearchCapabilities.CAPABILITIES;
```

Comment:
Hmm... I think async_search is going to need to "inherit" the same set of capabilities - or at least this one.